vllm/model_executor/layers/fused_moe/deep_gemm_moe.py

```diff
@@ -53,7 +53,9 @@ def _valid_deep_gemm(hidden_states: torch.Tensor, w1: torch.Tensor,
     _, K, N = w2.size()
     if not _valid_deep_gemm_shape(M, N, K):
         logger.debug_once(
-            "DeepGemm disabled: unaligned problem size. M: %s, N: %s, K: %s",
+            "DeepGemm disabled for this problem size. M: %s, N: %s, K: %s. "
+            "This log means we will fallback to triton "
+            "for this specific shape for further speed up.",
```

Comment:
The fallback in the small M case is for speedup, but the fallback if N or K is invalid is needed to prevent crashing. I think we should clarify this. cc @varun-sundar-rabindranath and @bnellnm who know more specifics

Reply:
Yep, that sounds good to me

Reply:
Fixed